maintainers/maintainer-list.nix

```diff
@@ -5848,6 +5848,12 @@
     githubId = 35892750;
     name = "Maxine Aubrey";
   };
+  maxhbr = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add maxhbr``.